dash_docs/chapters/dash_datatable/conditional_formatting/index.py

```diff
@@ -141,21 +141,20 @@
                     'textAlign': 'left'
                 },
 
+                {
+                    'if': {
+                        'state': 'active'  # 'active' | 'selected'
+                    },
+                    'backgroundColor': 'DodgerBlue',
+                    'opacity': 0.5,
+                    'border': '2px solid DodgerBlue'
```

Comment:
```suggestion
 ```suggestion
                   'backgroundColor': 'rgba(0, 116, 217, 0.3)'
```
```